docs/topics_5/web-openshift-troubleshoot.adoc

```diff
@@ -100,67 +100,67 @@ After attempting to deploy {WebNameTitle} on OpenShift, the Pod is unable to sta
 
 *What it means:*
 
-The resource quota for the OpenShift project has been met, and the Pod is unable to obtain the requested resources.
+The resource quota for the OpenShift project has been met, and the pod is unable to obtain the requested resources.
 
 *How to resolve it:*
 
-Perform either of the following:
+Do either of the following:
 
-* Increase the quota for the OpenShift project. For additional information, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges] in the OpenShift documentation.
+* Increase the quota for the OpenShift project. For additional information on OpenShift quotas, see link:{OpenShiftDocsURL}/applications/quotas/quotas-setting-per-project.html[Quotas and Limit Ranges] and link:{OpenShiftDocsURL}/nodes/clusters/nodes-cluster-limit-ranges.html[Setting Limit Ranges].
 * Reduce the requested resources for the {WebName} OpenShift project. It is recommended to have at least 2 CPUs and 4 GB of memory for the project.
 
 Once the request is within the available quota, attempt the deployment once again.
 
-.Pod takes longer than 600 seconds to become available
+.A pod takes longer than 600 seconds to become available
```

Comment:
I don't think you need such a long title. Besides, if the developers ever change the time-out limit, you will have to update this. Safer to say "Deployment pods time out"